tests/python/cli/test_cli.py

```diff
@@ -302,3 +302,16 @@ def test_can_control_organization_context(self):
         all_task_ids = list(map(int, self.run_cli("ls").split()))
         assert personal_task_id in all_task_ids
         assert org_task_id in all_task_ids
+
+    def test_auto_annotate(self, fxt_new_task: Task):
+        annotations = fxt_new_task.get_annotations()
+        assert not annotations.shapes
+
+        self.run_cli(
+            "auto-annotate",
+            str(fxt_new_task.id),
+            f"--function={__package__}.example_function",
+        )
```

Comment:
I suggest adding several other tests for different file placement, because we don't have a package when we call it from the actual CLI (in the tests we cheat a little).

Reply:
- A test that we can execute a function from a file/module in the current directory
- A test that we can execute a function from a file/module outside the current directory subtree, and which is not installed in the current system or python environment (e.g. `cvat-cli auto-annotate --function ../../my_cvat_model.py ...`

Reply:
I don't see the point in these tests to be honest. We don't do anything unusual with the import mechanics. If Python can import a module, then so can we (and vice versa).